common/workunit/workunit.cpp

```diff
@@ -1087,6 +1089,17 @@ class CDaliWorkUnit : public CPersistedWorkUnit
         VStringBuffer path("/GraphProgress/%s/%s", queryWuid(), graphName);
         return querySDS().connect(path, myProcessSession(), RTM_LOCK_WRITE|RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT);
     }
+    IPropertyTree *getGraphProgressTree() const
+    {
+        IRemoteConnection *conn = queryProgressConnection();
```

Comment:
That's what I was wondering (should have said).
Yes reload() would be the way to refresh it, if deemed necessary.
If it's something that happens a lot though, it would be a pity to keep reloading..
Ideally, it would be marked dirty and only reloaded then (e.g. via a subscription or by polling a changing id).